src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/appointments/AppointmentsService.kt

```diff
@@ -3,6 +3,7 @@ package uk.gov.justice.digital.hmpps.prisonertonomisupdate.appointments
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.kotlin.readValue
 import com.microsoft.applicationinsights.TelemetryClient
+import org.apache.commons.lang3.StringUtils.isBlank
```

Comment:
Kotlin has its own version of this - `isNullOrBlank` meaning that a 3rd party lib isn't required.

Reply:
Ah thanks - missed that one. Will use it in next commit